Lib/test/test_dataclasses.py

```diff
@@ -2926,23 +2926,54 @@ class C:
                 x: int
 
     def test_generated_slots_value(self):
-        @dataclass(slots=True)
-        class Base:
-            x: int
 
-        self.assertEqual(Base.__slots__, ('x',))
+        class Root:
+            __slots__ = {'x'}
+
+        class Root2(Root):
+            __slots__ = {'k': '...', 'j': ''}
```

Comment:
Shouldn't you be testing that 'k' and 'j' aren't added in a derived class?

Reply:
done.